pages/linux/binwalk.md

```diff
@@ -0,0 +1,28 @@
+# binwalk
+
+> Firmware Analysis Tool.
+> More information: <https://github.com/ReFirmLabs/binwalk>.
+
+- Scan a binary file:
+
+`binwalk {{path/to/binary}}`
+
+- Extract files from a binary, specifying the output directory:
+
+`binwalk --extract --directory {{output_directory}} {{path/to/binary}}`
+
+- Recursively extract files from a binary limiting the recursion depth to 2:
+
+`binwalk --extract --matryoshka --depth {{2}} {{path/to/binary}}`
+
+- Extract files from a binary with the specified file signature:
+
+`binwalk --dd '{{png image:png}}' {{path/to/binary}}`
+
+- Analyze entropy of a binary, saving the plot (with the same name as the binary and the PNG extension appended):
```

Comment:
Statement inside parentheses should be put outside them. And perhaps reworded.

Reply:
Perhaps a specific suggestion would be better, @zdroid?

What about this?
```suggestion
- Analyze the entropy of a binary, saving the plot with the same name as the binary and `.png` extension appended:
```